docs/sdk/migrations/6.0.0.md

```diff
@@ -0,0 +1,491 @@
+---
+sidebar_position: 3
+---
+
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Migrating from 5.x.x to 6.x.x
+
+## Breaking Changes & Important Updates
+
+The most significant change in v6 is the combination of `authSig` and `sessionSigs`. In most functions, the client-side generated `authSig` will no longer be accepted as an argument, as seen in commonly used functions like `executeJs` and `pkpSign`. Instead, it will be used to generate `sessionSigs` to authenticate with the Lit nodes.
+
+### Per-Package Changes
+
+#### `@lit-protocol/lit-node-client`
+
+##### executeJs
+
+- v5 - optional use of `authSig` or `sessionSigs`, but either must exist.
+- v6 - `sessionSigs` is strictly required
+
+```tsx
+// v5
+import { AuthSig, SessionSigsMap } from '@lit-protocol/types';
+
+const authSig: AuthSig = '...';
+const sessionSigs: SessionSigsMap = '...';
+
+const res = await litNodeClient.executeJs({
+	authSig?: authSig,
+	sessionSigs?: sessionSigs;
+	...
```

Comment:
we need to add `jsParams` as it is required for execution.

Reply:
I don't think we do, jsParams can be omitted if you want to run lit action as a serverless function. Correct me if I'm wrong here @DashKash54

Reply:
Are you using any params? If not why use jsParams?

Reply:
They are not required as I remember. I think its okay as its just showing the changes needed.